tools/cd_scripts/install_test.sh

```diff
@@ -22,46 +22,31 @@ gsutil cp  gs://gcsfuse-release-packages/version-detail/details.txt .
 curl http://metadata.google.internal/computeMetadata/v1/instance/name -H "Metadata-Flavor: Google" >> details.txt
 touch ~/logs.txt
 
-# uname can be x86_64 or aarch64
-uname=$(uname -m)
-if [[ $uname == "x86_64" ]]; then
-  architecture="amd64"
-elif [[ $uname == "aarch64" ]]; then
-  architecture="arm64"
-fi
-
-# Based on the os type(from vm instance name) in detail.txt, run the following commands to install apt-transport-artifact-registry
+# Based on the os type(from vm instance name) in detail.txt, run the following
+# commands to install gcsfuse.
 if grep -q ubuntu details.txt || grep -q debian details.txt;
 then
 #  For ubuntu and debian os
-    curl https://us-central1-apt.pkg.dev/doc/repo-signing-key.gpg | sudo apt-key add - && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-    # Here we used apt-transport-artifact-registry-unstable main as stable one does not supports arm64, will switch back to stable once they support.
-    echo 'deb http://packages.cloud.google.com/apt apt-transport-artifact-registry-unstable main' | sudo tee -a /etc/apt/sources.list.d/artifact-registry.list
-    sudo apt update
-    sudo apt install apt-transport-artifact-registry
-    echo "deb ar+https://us-apt.pkg.dev/projects/gcs-fuse-prod gcsfuse-$(lsb_release -cs) main" | sudo tee -a /etc/apt/sources.list.d/artifact-registry.list
-    sudo apt update
-
-    # Install released gcsfuse version
-    sudo apt install -y gcsfuse=$(sed -n 1p details.txt) -t gcsfuse-$(lsb_release -cs) |& tee -a ~/logs.txt
+    export GCSFUSE_REPO=gcsfuse-`lsb_release -c -s`
+    echo "deb https://packages.cloud.google.com/apt $GCSFUSE_REPO main" | sudo tee /etc/apt/sources.list.d/gcsfuse.list
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+    sudo apt-get update
+    # Install latest released gcsfuse version
+    sudo apt-get install -y gcsfuse
 else
 #  For rhel and centos
-    # arm64 machines only supports dnf
-    sudo yum -y install dnf
-    sudo dnf makecache
-    sudo yum -y install dnf-plugin-artifact-registry
-sudo tee -a /etc/yum.repos.d/artifact-registry.repo << EOF
-[gcsfuse-el7-x86-64]
-name=gcsfuse-el7-x86-64
-baseurl=https://asia-yum.pkg.dev/projects/gcs-fuse-prod/gcsfuse-el7-x86-64
+    sudo yum install fuse
+    sudo tee /etc/yum.repos.d/gcsfuse.repo > /dev/null <<EOF
+[gcsfuse]
+name=gcsfuse (packages.cloud.google.com)
+baseurl=https://packages.cloud.google.com/yum/repos/gcsfuse-el7-x86_64
 enabled=1
-repo_gpgcheck=0
 gpgcheck=1
+repo_gpgcheck=0
 gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
-    https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+      https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
```

Comment:
nit: remove extra space.

Reply:
I copy pasted the install command from https://cloud.google.com/storage/docs/gcsfuse-install because of which this extra space crept in. 😅 Going ahead with the merge.